database/metadata/cron_triggers.yaml

```diff
@@ -1 +1,17 @@
-[]
+- name: refresh_location_crashes_view
+  webhook: http://localhost:8080/v2/query
+  schedule: '0 * * * *'
```

Comment:
we could modify this so that it only runs during business hours. i think i'd prefer to have this run consistently so we don't have to try and remember what the table's schedule is 🤷